independent-publisher-connectors/Notion/apiDefinition.swagger.json

```diff
@@ -330,21 +425,223 @@
+                      "title": "Color"
+                    }
+                  },
+                  "description": "paragraph"
+                },
+                "heading_1": {
+                  "type": "object",
+                  "properties": {
+                    "rich_text": {
+                      "type": "array",
+                      "items": {
+                        "type": "object",
+                        "properties": {
+                          "text": {
+                            "type": "object",
+                            "properties": {
+                              "content": {
+                                "type": "string",
+                                "description": "Text content. This field contains the actual content of your text and is probably the field you'll use most often.",
+                                "title": "Content"
+                              }
+                            },
+                            "description": "text",
+                            "title": "Text"
+                          }
+                        }
+                      },
+                      "description": "rich_text",
+                      "title": "Heading 1"
+                    },
+                    "color": {
+                      "type": "string",
+                      "description": "Color of the block.",
+                      "title": "Color"
+                    }
+                  },
+                  "description": "heading_1",
+                  "x-ms-visibility": "advanced"
+                },
+                "heading_2": {
+                  "type": "object",
+                  "properties": {
+                    "rich_text": {
+                      "type": "array",
+                      "items": {
+                        "type": "object",
+                        "properties": {
+                          "text": {
+                            "type": "object",
+                            "properties": {
+                              "content": {
+                                "type": "string",
+                                "description": "Text content2. This field contains the actual content of your text and is probably the field you'll use most often.",
+                                "title": "Content"
+                              }
+                            },
+                            "description": "text",
+                            "title": "Text"
+                          }
+                        }
+                      },
+                      "description": "rich_text",
+                      "title": "Heading 2"
+                    },
+                    "color": {
+                      "type": "string",
+                      "description": "Color of the block.",
+                      "title": "Color"
+                    }
+                  },
+                  "description": "heading_2",
```

Comment:
@sekharmalla All the description values in the swagger are expected to be free from grammatical and spell errors. Please provide valid description values for all the properties.

Reply:
Updated proper descriptions for the swagger file.

Thanks.